Automattic/wc-calypso-bridge
# Add a workspaces section to our to-level package.json

### Changes proposed in this Pull Request:

This sets it up so that running `npm i` in the project root will also insall the dependencies defined by `package.json` in the `./scripts/` directory.



Related to #1096






### How to test the changes in this Pull Request:




1. Run the command `rm -rf ./node_modules ./scripts/node_modules` in the project root.
2. Run the command `npm i` in the project root.
3. Verify that there is a `node_modules/` directory in both the project root _and_ the `scripts/` directory.



### Other information:




### FOR PR REVIEWER ONLY:
